include/lsst/gauss2d/fit/transforms.h

```diff
@@ -7,6 +7,10 @@
 
 #include "lsst/modelfit/parameters.h"
 
+#ifndef pow10
```

Comment:
What is supposed to define `pow10`  as macro?
Is there some missing header like `math.h`?

Reply:
OK. `pow10` is removed in glibc 2.17 from `math.h` and is also a GNU extension
https://man7.org/linux/man-pages/man3/pow10.3.html
This still seems wrong to define it when the convenience macro is not defined.
Maybe just replace it with `pow`